src/android/ContentFilesystem.java

```diff
@@ -18,22 +18,19 @@ Licensed to the Apache Software Foundation (ASF) under one
  */
 package org.apache.cordova.file;
 
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.OutputStream;
-
-import org.apache.cordova.CordovaResourceApi;
-import org.json.JSONArray;
-import org.json.JSONException;
-import org.json.JSONObject;
-
 import android.content.ContentResolver;
 import android.content.Context;
 import android.database.Cursor;
 import android.net.Uri;
+import android.provider.DocumentsContract;
 import android.provider.MediaStore;
 import android.provider.OpenableColumns;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import org.apache.cordova.CordovaResourceApi;
+import org.json.JSONException;
+import org.json.JSONObject;
```

Comment:
Please keep the import statements separated into groups. Ideally I think they should follow the [Android style](http://source.android.com/source/code-style.html#order-import-statements), but not all of Cordova's Android code follows that now.